### PR TITLE
Add option to wish for an eagle

### PR DESCRIPTION
**Pull request by rajkk1**
Hello!

Thank you very much for the course, it was really instructive :)

I've added a new line to the story!

Cheers :D

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
Glad you enjoyed the course! @casey-collab, please review this.

File: english/explore-outside/journey-to-flaming-tower/walk/portal/wishes/eagle/eagle.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+With the wave of his hands, the little fellow summons an eagle! You both jump on the back of it and
```
Comment (casey-collab) [inline english/explore-outside/journey-to-flaming-tower/walk/portal/wishes/eagle/eagle.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/explore-outside/journey-to-flaming-tower/walk/portal/wishes/eagle/wishes.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+With the wave of his hands, the little fellow summons an eagle! You both jump on the back of it and
```
Comment (casey-collab) [inline english/explore-outside/journey-to-flaming-tower/walk/portal/wishes/eagle/wishes.md]: This file has also been added to a directory that already contained a page.

File: english/explore-outside/journey-to-flaming-tower/walk/portal/wishes/wishes.md [hunk 3]
```diff
@@ -2,4 +2,5 @@ The little red fellow kind of creeps you out
  but you ask him to accompany you to the tower
   because these wishes might just come in handy ...
 
-[Wish that you and him teleport to the tower](./teleport/teleport.md)
\ No newline at end of file
+[Wish that you and him teleport to the tower](./teleport/teleport.md)
+[Wish that you and him get an eagle to ride on](./eagle/eagle.md)
```
Comment (casey-collab) [inline english/explore-outside/journey-to-flaming-tower/walk/portal/wishes/wishes.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: english/explore-outside/journey-to-flaming-tower/walk/portal/wishes/eagle/wishes.md (hunk 2 above)
Comment (casey-collab) [inline english/explore-outside/journey-to-flaming-tower/walk/portal/wishes/eagle/wishes.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by rajkk1** [comment]
Sorry about that! I seemed to have added an extra file in my folder. I've removed it and also added a line between lines. Cheers!

**Comment by cbuckey-uda** [comment]
No problem, thanks for making the fixes! @casey-collab, please take another look.

**Comment by casey-collab** [comment]
Looks great! :penguin:

**Pull request merged**